### ISSUE TITLE
Compare Images with "Live Pictures"

### ISSUE THREAD

**Issue by RomBrz** [WinMerge/winmerge#1448]
I don't know if that enter on "bugs" category or just "enhancements", but i discover a behavior that i think that should not happen.

Just to explain: I use OneDrive to backup my phone's (Galaxy S10+) photos. And i use the "Motion Photos" (Apple's Live Photos) enabled by default, so each picture i take has a short video embedded in the photo file.

Today when i was doing a backup check (checking my phone photo file against the OneDrive photo file), WinMerge told me that some files was different. But when i double clicked it, WinMerge told me that the pictures are the same. So i copied both files to another folder to check then individually, when i saw the half of the size, i remember a "recent" feature that arrived at Samsung Phones to easily remove the Live Pictures from a single file (ex: if i take a photo from a document, i don't need the Live Picture to this single file to be stored, so i can take the photo and then erase the Live Picture).

So my suggestion is on something to check if the Live Picture exists to make WinMerge more clear about what is the diference between files (in this case: WinMerge tell that the binary files are different, but when double clicked it tell that is the same, because the photo itself is the same, the difference rellie on the existence of the "video part" of the file that was missing on the cloud file).

(Using 2.16.22.0 x64 version)

**Comment by Gitoffthelawn** [comment]
@RomBrz Quick question: Do the Samsung Galaxy S10/S10+ devices have integrated support for "Motion Photos"?

**Comment by RomBrz** [comment]
@Gitoffthelawn yes. Since Galaxy S7 the flagships has native support for Motion Photos:

Motion Photo is available on Galaxy S20, S20+, S20 Ultra, Z Flip, Note10, Note10+, S10e, S10, S10+, Fold, Note9, S9, S9+, Note8, S8, S8+, S7, and S7 edge. (https://www.samsung.com/global/galaxy/what-is/motion-photo/)

Seems like Motion Photos is available since 2016, but is not a default option (you have to enable it) (source: https://www.cnet.com/tech/mobile/what-you-need-to-know-about-motion-photo-on-the-galaxy-s7/)

In this picture (https://r1.community.samsung.com/t5/image/serverpage/image-id/4545984i598F5C911BC115EB/image-size/large?v=v2&px=999) the Motion Photos can be enabled on the second button from right to left (between the "3:4" button and the magic stick).

**Comment by Gitoffthelawn** [comment]
@RomBrz Wow... that's an incredibly helpful answer to my little question.  Thank you so very much!
